test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/ValidationVocabularyTests.cs

```diff
@@ -86,7 +86,7 @@ public void TestValidationVocabularyModel()
   <Term Name=""AllowedValues"" Type=""Collection(Validation.AllowedValue)"" AppliesTo=""Property Parameter TypeDefinition"" Nullable=""false"">
     <Annotation Term=""Core.Description"" String=""A collection of valid values for the annotated property, parameter, or type definition"" />
   </Term>
-  <Term Name=""MultipleOf"" Type=""Edm.Decimal"" AppliesTo=""Property Parameter Term"" Nullable=""false"" Scale=""Variable"">
+  <Term Name=""MultipleOf"" Type=""Edm.Decimal"" AppliesTo=""Property Parameter Term"" Nullable=""false"" Scale=""variable"">

```

Comment:
Could we add a  test for the behavior of the capital V ?

Reply:
For what it's worth scale should not have a capital V.. the V should be in lower case.. Just confirming whether we need to support capital V for backward compatibility..